.github/workflows/update_instrumentation_dependencies.yaml

```diff
@@ -1,39 +1,42 @@
-name: Check for new java agent
+name: Check for new instrumentation versions
 
 on:
   schedule:
     - cron: "45 */4 * * *"
   workflow_dispatch:
 
 env:
-  JAVA_REPO: ghcr.io/signalfx/splunk-otel-java/splunk-otel-java
   VALUES_YAML: helm-charts/splunk-otel-collector/values.yaml
-  LATEST_API: https://api.github.com/repos/signalfx/splunk-otel-java/releases/latest
+  LATEST_API: https://api.github.com/repos/signalfx/splunk-otel-{lang}/releases/latest
 
 jobs:
-  maybe_update_java:
+  maybe_update:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        language: ['java'] # Add other languages here
```

Comment:
We might be able to have automation extract the list value for this line, will add in a future PR if possible.